test/functionalTest/cases/1308_subscription_cache/cache_reset_from_db.test

```diff
@@ -0,0 +1,667 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Possible Bug in Subscription Cache when reloading from database
+
+--SHELL-INIT--
+dbInit CB
+dbInit ftest-t1
```

Comment:
What?
CB is also not an env var.
There are NO env vars as input to dbInit. 
We have never had that.
And frankly, what on earth would we gain by hiding the string "ftest-t1" in a variable?